puppet/modules/quickstack/manifests/neutron/plugins/plumgrid.pp

```diff
@@ -0,0 +1,15 @@
+# Common class used by Plumgrid Neutron Plugin.
+class quickstack::neutron::plugins::plumgrid (
```

Comment:
I don't see this class calls/included anywhere in this patch, am I missing something?